docs/getting-started/full-dotnet.rst

```diff
@@ -0,0 +1,102 @@
+.. include:: /stub-topic.txt
+
+Getting Started on Full .NET (Console, WinForms, WPF, etc.)
+==========================================
+
+In this walkthough, you will build a console application that performs basic data access using Entity Framework.
+
+In this article:
+	- `Ensure NuGet 2.8.5 or later`_
+	- `Create a new project`_
+	- `Install Entity Framework`_
+	- `Create your model`_
+	- `Create your database`_
+	- `Use your model`_
+	
+`View this article's samples on GitHub <https://github.com/aspnet/EntityFramework.Docs/tree/master/docs/getting-started/full-dotnet/sample>`_.
+
+.. note::
+    This walkthough uses EF 7.0.0-beta4 which is the latest pre-release available on NuGet.org.
+
+    You can find nightly builds of the EF7 code base hosted on https://www.myget.org/F/aspnetvnext/api/v2/ but the code base is rapidly changing and we do not maintain up-to-date documentation for getting started.
+
+Ensure NuGet 2.8.5 or later
+---------------------------
+
+Installing EF7requires NuGet 2.8.5 (or higher). Make sure you restart Visual Studio after installing the update.
+
+- **Visual Studio 2015** - No updates needed, a compatible version of NuGet is included.
+- **Visual Studio 2013** - `Install the latest NuGet for VS2013 <https://visualstudiogallery.msdn.microsoft.com/4ec1526c-4a8c-4a84-b702-b21a8f5293ca>`_.
+- **Visual Studio 2010 and 2012** - `Install the latest NuGet for VS2012 and 2010 <https://visualstudiogallery.msdn.microsoft.com/27077b70-9dad-4c64-adcf-c7cf6bc9970c>`_.
+
+.. note::
+    NuGet version numbers can be confusing, while the required release is branded 2.8.5 the product version of the extension is 2.8.60318.xxx.
+
+Create a new project
+--------------------
+
+* Open Visual Studio (this walkthough uses 2015 but you can use any version from 2010 onwards)
+* :menuselection:`File --> New --> Project...`
+* From the left menu select :menuselection:`Templates --> Visual C# --> Windows`
+* Select the **Console Application** project template
+* Ensure you are targetting .NET 4.5.1 or later
+* Give the project a name and click **OK**
+
+Install Entity Framework
+----------------------------------------
+To use EF7 you install the package for the database provider(s) you want to target. This walkthrough uses SQL Server. For a list of available providers see :doc:`/providers/index`.
```

Comment:
We have talked about meta packages and what people will need to install to get things working and I wanted to say I think installing these two packages is a good experience. I don't think we need an additional meta package on top of this experience.